File: 백준/Bronze/2562. 최댓값/최댓값.js
let arr = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n').map(x=>parseInt(x));
let max=arr[0];
let idx=1;
for(let i=0;i<9;i++){
    if(arr[i]>max){
        max=arr[i];
        idx=i+1;
    }
}
console.log(max);
console.log(idx);

File: 백준/Bronze/2566. 최댓값/최댓값.py
arr=[]
for i in range(0,9):
    arr.append(list(map(int,input().split())))
max=0
x=1
y=1
for i in range(0,9):
    for j in range(0,9):
        if arr[i][j]>max:
            max=arr[i][j]
            x=i+1
            y=j+1
print(max)
print(x,y)